development/compiling/compiling_with_script_encryption_key.rst

```diff
@@ -37,14 +37,17 @@ Step by step (Linux)
    You can generate the key without redirecting the output to a file, but
    that way you can minimize the risk of exposing the key.
 
-
 2. Set this key as environment variable in the console that you will use to
    compile Godot, like this:
 
-   ::
+   .. tabs::
+    .. code-tab:: bash Linux/macOS
 
        export SCRIPT_AES256_ENCRYPTION_KEY="your_generated_key"
 
+    .. code-tab:: bat Windows
+
+       set SCRIPT_AES256_ENCRYPTION_KEY="your_generated_key"
```

Comment:
Doesn't `set` use this syntax?

```suggestion
       set SCRIPT_AES256_ENCRYPTION_KEY your_generated_key
```

Reply:
Always trouble with double quotes in examples. 🙂

You are right that the quotes would be included here, yet `=` is still required according to [set syntax](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/set_1#syntax). So I'll just remove the quotes then.